app/plugin_api.go

```diff
@@ -81,6 +81,11 @@ func (api *PluginAPI) GetConfig() *model.Config {
 	return api.app.GetSanitizedConfig()
 }
 
+// GetUnsanitizedConfig fetches the currently persisted config without removing sensible configuration
```

Comment:
Think i like your comment better ;-) `// GetUnsanitizedConfig gets the configuration for a system admin without removing secrets.`